lib/twilio/device.ts

```diff
@@ -972,7 +963,7 @@ class Device extends EventEmitter {
       maxAverageBitrate: this._options.maxAverageBitrate,
       preflight: this._options.preflight,
       rtcConstraints: this._options.rtcConstraints,
-      shouldPlayDisconnect: () => this._enabledSounds.disconnect,
+      shouldPlayDisconnect: () => this._audio ? this._audio.disconnect() : defaultEnabledSounds.disconnect,
```

Comment:
We should check if these members are defined, i.e. `this._audio.disconnect` might be undefined depending on the state of the `AudioHelper` and we might try to invoke it.